### PR TITLE
US011 Checking the presence of the state of a single CE in details

### PR DESCRIPTION
**Pull request by jcox-dev**
On hold until [UI changes](https://github.com/ONSdigital/response-operations-ui/pull/49) have gone in.

### CODE REVIEW

**Comment by jcox-dev** [comment]
[US011 Display State of a Collection Exercise [M]](https://trello.com/c/kJO9QkLd/29-us011-display-state-of-a-collection-exercise-m)

File: acceptance_tests/features/steps/view_collection_exercise_details_state.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+@given('the 1803 collection exercise for the QBS survey has been created')
+def qbs_1803_exists(_):
+    pass
+
+
+@given('the internal user navigates to the collection exercise details page for QBS 1803')
+@when('the internal user navigates to the collection exercise details page for QBS 1803')
+def internal_user_views_2017_bres_collection_exercise(_):
```
Comment (tej99) [inline acceptance_tests/features/steps/view_collection_exercise_details_state.py]: This is already being defined in view_collection_exercise_details.py can get rid of it.

File: acceptance_tests/features/steps/view_collection_exercise_details_state.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 1803 collection exercise for the QBS survey has been created')
+def qbs_1803_exists(_):
```
Comment (tej99) [inline acceptance_tests/features/steps/view_collection_exercise_details_state.py]: This is already being defined in view_collection_exercise_details.py can get rid of it.

File: acceptance_tests/features/steps/view_collection_exercise_details_state.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+@given('the internal user navigates to the collection exercise details page for QBS 1803')
+@when('the internal user navigates to the collection exercise details page for QBS 1803')
+def internal_user_views_2017_bres_collection_exercise(_):
+    collection_exercise_details.go_to('QBS', '1803')
+
+
+@then('the user is able to view the status of the collection exercise')
+def ce_details_state_is_displayed():
```
Comment (tej99) [inline acceptance_tests/features/steps/view_collection_exercise_details_state.py]: This needs a param. 
def ce_details_state_is_displayed(_):

**Comment by tej99** [comment]
Can we update acceptance_tests/features/view_collection_exercises_data.feature file line 18
| 1806   | 15 June 2018 | Created |   ==>  | 1806   | 15 June 2018 | Scheduled |
 and get remove the @skip tag?

**Comment by benjefferies** [comment]
What's the difference between this and https://github.com/ONSdigital/ras-integration-tests/pull/35/files? Seems like one will supersede the other?

File: acceptance_tests/features/view_collection_exercises_data.feature [hunk 4]
```diff
@@ -16,7 +15,7 @@ Feature: View Collection Exercise
     And the internal user can view all collection exercises for QBS
       | period | shown_to_respondent_as | status  |
       | 1803   | 9 March 2018           | Created |
```
Comment (tej99) [inline acceptance_tests/features/view_collection_exercises_data.feature]: Looks like 1803 events are also being loaded as part of make setup. so need to change Created ==> Scheduled for period 1803.

**Pull request merged**